Migrate SAML OneLogin configuration article

encima opened the pull request:
# What changed, and why it matters

Depends on https://github.com/aiven/devportal/pull/1204 before merging. Added the OneLogin article and elaborated on text to substitute screenshots. Perhaps 2 screenshots of the aiven console and onelogin app would be beneficial but I am not sure.

Please check the formatting of values and field names in particular, I tried to stay consistent with existing entries but I am still not sure if it is right!

Addresses https://github.com/aiven/devportal/issues/825

ftisiot commented:
Hi @encima! Good article, I changed the structure slightly to be more in line with devportal, could you please check? Also there's a missing image, do you still think it's needed? 

The rest is ok from my side, once you validate from your side I'll ask a second pair of eyes on the wording and then merge

encima commented:
> 

You are right, the image is useful but not necessary (especially when the design has changed quite a bit). Removed and added an explanation. The navigation reshuffle makes sense also, thank you!

ftisiot reviewed (approved):
LGTM
